client/src/_shared/LoggedInLayout.js

```diff
@@ -21,6 +23,34 @@ export function LoggedInLayout({ children, title }) {
     history.push('/login')
   }
 
+  const changeLanguage = lang => {
+    setLanguage(lang)
+    i18n.changeLanguage(lang)
+  }
+
+  const mobileMenu = (
+    <Menu>
+      {language === 'es' ? (
```

Comment:
See https://github.com/pieforproviders/pieforproviders/pull/472/files#r511680452. We can use `i18n.language` here.

Reply:
cool! good call.